libs/domain/picking/warehouse-assignment/warehouse-assignment.component.ts

```diff
@@ -0,0 +1,58 @@
+import { resolve } from '@spryker-oryx/di';
+import { WarehouseUserAssignmentsService } from '@spryker-oryx/picking';
+import { RouterService } from '@spryker-oryx/router';
+import { asyncState, i18n, Size, valueType } from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
+import { repeat } from 'lit/directives/repeat.js';
+import { tap } from 'rxjs';
+import { styles } from './warehouse-assignment.styles';
+
+export class WarehouseAssignmentComponent extends LitElement {
+  static styles = styles;
+
+  protected warehouseUserAssignmentsService = resolve(
+    WarehouseUserAssignmentsService
+  );
+
+  protected routerService = resolve(RouterService);
+
+  @asyncState()
+  protected locations = valueType(
+    this.warehouseUserAssignmentsService.getList()
+  );
+
+  protected onSelect(assignmentId: string): void {
+    this.warehouseUserAssignmentsService
+      .activateAssignment(assignmentId)
+      .pipe(
+        tap(() => {
+          this.routerService.navigate('/');
+        })
+      )
+      .subscribe();
+  }
+
+  protected override render(): TemplateResult {
+    return html`
```

Comment:
I believe we should not render anything when `this.locations` is falsy or empty

Reply:
I believe we need some fallback text instead. I will contact Nuwan for this case